### ISSUE TITLE
AO around the hair/character not working

### ISSUE THREAD

**Issue by jinhyung2003** [ruyo/VRM4U#65]
Hi. I am trying to accomplish the AO shown in the documentation (on ray tracing part). However it seems to not work despite checking that AO is turned on.

![example](https://user-images.githubusercontent.com/19562627/113577862-e77fc480-965c-11eb-81fb-0c0db9300500.PNG)

This is what I like to accomplish,

![캡처](https://user-images.githubusercontent.com/19562627/113577824-d636b800-965c-11eb-8bb6-b960d014ed49.PNG)

however, following the documentation, it just doesn't work.

**Comment by jinhyung2003** [comment]
The version I'm using is 4.26 btw...

**Comment by ruyo** [comment]
Hi.
In order to use AO, you need to use Lit mode or SSS mode.
You are probably using the default Unlit mode as an import option.

Please refer to this page to change the material mode.
You can change the mode when importing or using the AssetUtil actor.
https://d3etapyazx4hzgaanfq2hik3j4--ruyo-github-io.translate.goog/VRM4U/01_look/

Check the lighting settings as well.
https://d3etapyazx4hzgaanfq2hik3j4--ruyo-github-io.translate.goog/VRM4U/02_sss/

Thanks.

**Comment by jinhyung2003** [comment]
Thanks for the response.
However, changing Lit made another problem as following:

![894573473589](https://user-images.githubusercontent.com/19562627/113586019-f15af500-9667-11eb-8a6e-268edd8635d2.PNG)

First off, when I tried to change the mesh from unlit to lit through AssetUtil, but it won't apply the change. I also tried SS but neither worked. Since my project was full dynamic lightning, I changed back to static light but no difference has made. I'll try updating the plugin to the newest version.

**Comment by jinhyung2003** [comment]
+ Import option didn't worked either, showing as same as above

**Comment by jinhyung2003** [comment]
Updated the plugin and still shows the same.
It seems AssetUtil won't let me change the lit, and import option is still not working

**Comment by ruyo** [comment]
Are the previews and thumbnails of SkeletalMesh looking correct?
Try placing the model on a new map at the same time to see how it looks.
My guess is that some parameter has been changed in MToonMaterialSystem.

**Comment by jinhyung2003** [comment]
I tried on new maps and there is no difference. Hmmm......

**Comment by jinhyung2003** [comment]
Yes. The thumbnails are looking correct, but not the skeletal mesh.

**Comment by jinhyung2003** [comment]
![333](https://user-images.githubusercontent.com/19562627/113589733-b3ac9b00-966c-11eb-8379-4616df579158.PNG)

![444](https://user-images.githubusercontent.com/19562627/113589767-be673000-966c-11eb-9f81-f7560c15431e.PNG)

![8978](https://user-images.githubusercontent.com/19562627/113589828-d2129680-966c-11eb-87c2-f5d21d056a5b.PNG)

This is what happens when I import as lit.

**Comment by ruyo** [comment]
I have no problem with the latest version.

To test it, create a NewLevel (Epic's official NewLevel) and place it there. Will there be a problem here too?
Set the material parameters and lighting to standard and check.

![image](https://user-images.githubusercontent.com/6472835/113594517-c7f39680-9672-11eb-8fc1-7e4f4acb106c.png)

**Comment by jinhyung2003** [comment]
Found an issue.

I've created a new project and made new "default" level, and found out that both Lit and SSS only worked when I allowed static light and going full dynamic lightning caused an issue. Checking the "Allow static Lightning" in rendering section solved the issue.

Since my game required full dynamic lightning, I guess Unlit is the only option... but anyway, thanks for the help.
Additionally, will it be possible for lit and sss, or AO to work while the lightning is full dynamic?
